lib/ES/Util.pm

```diff
@@ -77,7 +77,8 @@ sub build_chunked {
         my $asciidoc_dir = dir('resources/asciidoc-8.6.8/')->absolute;
         eval {
             $output = run(
-                'asciidoctor', '-v',
+                'asciidoctor', '-v', '--trace',
```

Comment:
`--trace` gives us a traceback when there is an error which is important if we're going to debug these things....